README.md

```diff
@@ -160,16 +208,16 @@ The Configurations Table will be used in conjunction with logging data in [Mothe
 
```

Comment:
We also probably never want to delete gardens, sensors, or actuators overall. Once we have logs that link back to these entities we don't want to be deleting them